### PR TITLE
test: set up happo

### LINKED ISSUE #563: Add Happo to CI
### Description

As a developer
I want to add screenshot testing to the project
So that I can check if changes to components' appearance are intentional or not
### Tasks (How)
- [ ] install happo
- [ ] set up a github action to run happo comparisons in CI

### PR DESCRIPTION
**Pull request by BCerki**
Fixes #563

## Notes

- To view the happo screenshots (both bcgov and button), click on the `Details` link in the Happo check section
- From there, you can approve the screenshots with the dropdown in the left sidebar (they can all be approved because this is the first time we're generating them)

### CODE REVIEW

File: docs/developer-guide.md [hunk 1]
```diff
@@ -37,6 +37,15 @@ To run happo tests:
 - create an `.env` file based on `env.example` and contact Button Inc. for credentials
 - `cd` into the desired package and run `yarn happo run`
 
+## Installing packages
+
+To install a new package
+
+- in the root directory, `yarn add [packageName] -W`
+- navigate to the root `package.json` and copy the package's line
+- navigate to the themes' `package.json` and paste the package's line
```
Comment (joshgamache) [inline docs/developer-guide.md]: Would using `yarn workspace "@button-inc/[theme]" add [packageName]`, yarn's built-in workspaces (which Lerna leverages) work here?

File: docs/developer-guide.md [hunk 2]
```diff
@@ -17,17 +24,25 @@
...
-- Run `bash .git/hooks/pre-commit` from repo to execute pre-commit on staged files without commiting
+#### Unit tests
 
-What are [pre-commit hooks](https://pre-commit.com/)?
+To run unit tests in all packages, run `yarn test:jest`
 
+#### Happo screenshots
 
-### Test
+To run generate happo screenshots locally:
```
Comment (joshgamache) [inline docs/developer-guide.md]:
```suggestion
To generate happo screenshots locally:
```

File: docs/developer-guide.md (hunk 1 above)
Comment (BCerki) [reply docs/developer-guide.md]: I think you might be looking at an old commit; this turned out not to be necessary. But thanks for the info! I didn't know that about workspaces

**Review by joshgamache — DISMISSED** [review]
Code and results all look good! I was able to generate the screenshots locally and they showed up the CI/CD as well. Will be nice to have those as easy identifiers going forward. Nice work getting it all integrated, looks like the code belies the complexity. 🌮

**Review by joshgamache — APPROVED** [review]

**Pull request merged**